indico/htdocs/js/indico/jquery/global.js

```diff
@@ -67,12 +70,30 @@ $(document).ready(function() {
         $target.attr('data-qtip-oldtitle', title);
         $target.removeAttr('title');
 
+        var position = $(this).data('qtip-position');
+        if (position == "left") {
+            var positionOptions = {
```

Comment:
don't use `var foo` more than once in the same scope. Better put `var positionOptions` before the if blocks and then just assign it in there.